fabric/dataclient.go

```diff
@@ -387,104 +387,22 @@ func convertOne(fg *Fabric) ([]*eskip.Route, error) {
 			lbEndpoints: endpoints,
 		}
 
-		defaultPrivileges := []interface{}{
+		defaultScopePrivileges := []interface{}{
```

Comment:
Why slice of interface and not slice of concrete type? What other types are expected there? Same Q for `defaultAllowList`, `privs` and `allowedServices` below.

Reply:
yes because of CreateFilter() in skipper has args set as `[]interface{}`, converting later would pollute the code later.

Reply:
Because they will be fed as filter args for Skipper, which obviously requires a slice of empty interface.